frontend/src/index.tsx

```diff
@@ -6,6 +6,9 @@ import { BrowserRouter as Router } from 'react-router-dom';
 import { App } from 'components/app/app';
 import { store } from 'store/store';
 
+import { ToastContainer } from 'react-toastify';
```

Comment:
```suggestion
import { Toast } from 'components/common/common';
```
create a common component for it